client/web/src/repo/blob/own/FileOwnershipPanel.tsx

```diff
@@ -36,6 +36,7 @@ export const FileOwnershipPanel: React.FunctionComponent<OwnershipPanelProps & T
             currentPath: filePath,
         },
     })
+    const [makeOwnerError, setMakeOwnerError] = React.useState<Error | null>(null)
```

Comment:
nit: style-wise to have all imports "unqualified" (or how is it called in js), you can extend the import (`import { useEffect, useState } from 'react'` and make this change:
```suggestion
    const [makeOwnerError, setMakeOwnerError] = useState<Error | null>(null)
```